clients/drcachesim/tools/view.cpp

```diff
@@ -604,13 +605,16 @@ view_t::parallel_shard_memref(void *shard_data, const memref_t &memref)
         disasm.insert(newline, str.str());
     }
     // Put our prefix on raw byte spillover, and skip the other columns.
-    newline = disasm.find('\n');
-    if (newline != std::string::npos && newline < disasm.size() - 1) {
+    size_t pos = 0;
+    newline = disasm.find('\n', pos);
+    while (newline != std::string::npos && newline < disasm.size() - 1) {
```

Comment:
This change goes away if it all fits on one or 2 lines?

Reply:
Yes, it would.

Reply:
Not relevant to this PR anymore. Fixed in view tool PR.